src/Binance/Entry.h

```diff
@@ -21,7 +21,7 @@ class Entry: public CoinEntry {
     virtual bool supportsJSONSigning() const { return true; }
     virtual std::string signJSON(TWCoinType coin, const std::string& json, const Data& key) const;
 
-    virtual HashPubkeyList preImageHashes(TWCoinType coin, const Data& txInputData) const;
+    virtual Data preImageHashes(TWCoinType coin, const Data& txInputData) const;
```

Comment:
`HashPubkeyList` is not used everywhere?

Reply:
In bitcoin, we still use it